src/App/DoubleExpression.java

```diff
@@ -0,0 +1,8 @@
+package App;
+
+public class DoubleExpression {
+    public static boolean doubleExpression(double a, double b, double c) {
+        final double precision = 1e-4;
```

Comment:
make it static final
add access modifier
read about constant names
https://mate-academy.github.io/style-guides/java/java.html#s5-naming

Reply:
why did not you  resolve this comment?

Reply:
Sorry, something wrong with this PR, this file was updated according to comments: https://github.com/AristarkhV/Homework/pull/1/commits/dda3c2a8c074ded37414200cb1d02f3372616b3d

Reply:
I created a new PR, it is more convenient to view https://github.com/AristarkhV/Homework/pull/3/commits/5cca2386ca3bf4b2c5c70895e507df2cacf36942

Reply:
> why did not you resolve this comment?

Sorry, something wrong with this PR, this file was updated according to comments: dda3c2a
I created a new PR, it is more convenient to view 5cca238